src/ensembl/src/content/app/entity-viewer/shared/helpers/transcripts-sorter.ts

```diff
@@ -127,6 +127,8 @@ type GeneViewSortableTranscript = IsProteinCodingTranscriptParam &
   GetSplicedRNALengthParam & {
     slice: SliceWithOnlyLength;
     spliced_exons: unknown[];
+    stable_id?: string;
+    name?: string;
```

Comment:
These fields do not take part in the sorting, right? Then they shouldn't be here. This type describes a minimum necessary set of fields to be able to sort the transcripts.

Reply:
✅